src/platform/qt/Window.cpp

```diff
@@ -1429,6 +1429,20 @@ void Window::setupMenu(QMenuBar* menubar) {
 	}
 	m_config->updateOption("fastForwardRatio");
 
+	addGameAction(tr("Fast forward speed increment"), "fastForwardUp", [this] {
+		float oldRatio = 0.0;
+		oldRatio = m_config->getOption("fastForwardRatio", oldRatio).toFloat();
+		float newRatio = oldRatio + 1.0;
+		m_config->setOption("fastForwardRatio", QVariant(newRatio));
```

Comment:
You can do this in two lines by using only one variable and adding `1.0f` inline. There's no need for two variables.

Also did you test this? I don't know if this would actually work since it doesn't reload the configuration.

Reply:
I reduced the number of lines to zero. Yes, I tested it and it works.

Reply:
I reduced the number of lines to two. Yes, I tested it and it works.